core/src/test/scala/scalacache/PackageObjectSpec.scala

```diff
@@ -20,25 +20,25 @@ class PackageObjectSpec extends FlatSpec with Matchers with BeforeAndAfter with
   behavior of "#get"
 
   it should "call get on the cache found in the ScalaCache" in {
-    scalacache.get("foo")
+    scalacache.get[String]("foo")
```

Comment:
Yeah, unfortunately, it looks like having the implicitly resolved `Codec[A]` (doesn't seem to matter if it is done via a context bound or separate implicit param) seems to require us to state the type param when calling the method. Added a note to the changelog.